packages/commonwealth/client/scripts/views/components/sidebar/CollapsableSidebarButtonUnscoped.tsx

```diff
@@ -0,0 +1,33 @@
+import 'components/sidebar/collapsable_button.scss';
+import React, { useState } from 'react';
+import useSidebarStore from '../../../state/ui/sidebar/index';
+import { CWIconButton } from '../component_kit/cw_icon_button';
+
+export const CollapsableSidebarButtonUnscoped = () => {
```

Comment:
We do not need two separate components if they are pretty much exactly the same. The differences might be handled with props. Please unify CollapsableSidebarButtonScoped and CollapsableSidebarButtonUnscoped by passing type scoped/unscoped

Reply:
Its just a little bit more difficult to follow if we put everything in a single component. This is because there are some funky stuff with the hidden-box that needs to be accounted for in scoped pages.

Anyways, unified the components